src/connector/connector-client.ts

```diff
@@ -37,6 +39,7 @@ export const ConnectorClient = (input: {
   createWebRtcSubjects?: () => WebRtcSubjectsType
   createSignalingSubjects?: () => SignalingSubjectsType
   subjects?: ConnectorClientSubjects
+  dependencies: { WebRTC: WebRTC; WebSocket: WebSocket }
```

Comment:
from clients perspective this is a breaking change. Can you make `dependencies` optional?